target/linux/ath79/image/mikrotik.mk

```diff
@@ -62,3 +62,11 @@ define Device/mikrotik_routerboard-wapr-2nd
   IMAGE_SIZE := 16256k
 endef
 TARGET_DEVICES += mikrotik_routerboard-wapr-2nd
+
+define Device/mikrotik_routerboard-mapl-2nd
```

Comment:
Please apply proper alphabetic sorting.

Reply:
Should I reorder just the `mikrotik.mk` or also `board.d/*` and `hotplug.d/*` files?

Reply:
Well, all of them should be sorted alphabetically ...

Reply:
Sorted.